src/test/java/org/project/springweb/controller/BookControllerTest.java

```diff
@@ -0,0 +1,151 @@
+package org.project.springweb.controller;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.springframework.security.test.web.servlet.setup.SecurityMockMvcConfigurers.springSecurity;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.project.springweb.dto.book.BookDto;
+import org.project.springweb.dto.book.CreateBookRequestDto;
+import org.project.springweb.exception.EntityNotFoundException;
+import org.project.springweb.util.TestUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+
+@Sql(scripts = {"classpath:database/categories/add-three-categories-to-categories.sql",
+        "classpath:database/books/add-three-books.sql"},
+        executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
```

Comment:
minor comment, Sql.ExecutionPhase.BEFORE_TEST_METHOD by defualt, so you can remove it